instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/messaging/MessagingAttributesExtractor.java

```diff
@@ -78,6 +80,11 @@ public void onStart(AttributesBuilder attributes, Context parentContext, REQUEST
     if (operation == RECEIVE || operation == PROCESS) {
       internalSet(attributes, SemanticAttributes.MESSAGING_OPERATION, operation.operationName());
     }
+
+    String messagePayload = getter.messagePayload(request);
+    if (messagePayload != null && messagePayload.length() <= HS_MAX_PAYLOAD_SIZE) {
```

Comment:
String length confused with byte length?

Reply:
No, I intentionally didn't want to add logic here